src/commands/install.yml

```diff
@@ -1,21 +1,21 @@
 description: >
-  Install a specific version of MATLAB. (If you use this command to install transformation products,
-  such as MATLAB Coder and MATLAB Compiler, the command does not automatically license such products for you.)
+  Install a specific version of MATLAB.
```

Comment:
Because this is so short, do you want to add some details? Here is what we have for GitHub Actions:

The Setup MATLAB action enables you to run MATLAB® code and Simulink® models with a specific version of MATLAB. When you specify this action as part of your workflow, the action sets up your preferred MATLAB release (R2021a or later) on a Linux®, Windows®, or macOS runner. If you do not specify a release, the action sets up the latest release of MATLAB. As part of the setup process, the action prepends MATLAB to the PATH system environment variable.

Reply:
I am ok with adding more detail here, but some of this information is also repeated down at each parameters. How do you think we should handle that?

Reply:
I feel having a little more up there (and some redundancy) is helpful. Not sure we need all the paragraph from setup-matlab, but whatever could be useful to the user at a first glance.